lux/test/lux/storage/job_opening_storage_test.exs

```diff
@@ -0,0 +1,176 @@
+defmodule Lux.Storage.JobOpeningStorageTest do
+  use ExUnit.Case, async: true
```

Comment:
In the main `Lux` project we have `UnitCase` and `IntegrationCase`, so that only the right tests run when we call something like `mix test.unit`. This is probably not necessary for `LuxApp`.